<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>二维码生成预约编码</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>32</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>渠道编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="number_lineEdit">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>32</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入渠道编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>60</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>条线名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="stripling_name_lineEdit">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>60</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>88</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>编码首字母：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_initial">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>88</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>116</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>编码数量：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_num">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>116</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>144</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>产品规格：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_specification">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>144</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>172</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>产品名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_product">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>172</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_channel">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>200</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>200</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>渠道：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
